src/component/dataZoom/AxisProxy.js

```diff
@@ -481,8 +481,8 @@ function setAxisModel(axisProxy, isRestore) {
     var useOrigin = isRestore || (percentWindow[0] === 0 && percentWindow[1] === 100);
 
     axisModel.setRange(
-        useOrigin ? null : +valueWindow[0].toFixed(precision),
-        useOrigin ? null : +valueWindow[1].toFixed(precision)
+        (useOrigin || valueWindow[0] === null) ? null : +valueWindow[0].toFixed(precision),
+        (useOrigin || valueWindow[1] === null) ? null : +valueWindow[1].toFixed(precision)
```

Comment:
Currently, The items in `valueWindow` should not be `null` by design.
I found that you are using the previous version of echarts, where the items in `valueWindow` might be `null`. 
This issue does not exist in the current version of echarts. 

Please double check the code (v4.4.0):
https://github.com/apache/incubator-echarts/blob/0e065ceba1d3db4c29a927fb94bbfb70b26ba6a9/src/component/dataZoom/AxisProxy.js#L192

Reply:
yes, this was on 4.2.1. Unfortunately we cannot really test with the latest version with a time axis since 4.4 and 4.5 break dataZoom as described in #11679. Will report back once that's sorted. Thanks all.